conf/routes

```diff
@@ -56,7 +56,7 @@ GET         /photo/encounter/:id                               @femr.ui.controll
 GET         /triage/:id                                        @femr.ui.controllers.TriageController.indexPopulatedGet(id: Integer)
 GET         /triage                                            @femr.ui.controllers.TriageController.indexGet()
 POST        /triage                                            @femr.ui.controllers.TriageController.indexPost(id: Integer)
-POST        /triage/:id                                        @femr.ui.controllers.TriageController.deletePatientPost(id: Integer)
+POST        /triage/:id                                        @femr.ui.controllers.TriageController.deletePatientPost(id: Integer, deleteByUserID: Integer)
```

Comment:
Hey Vivek - @ao3742 

Glad you were able to implement @kevinzurek's changes. Make sure you commit those changes and push them to the branch on your fork to update the pull request. All you need to do is push to your remote's fork (origin) and this pull request will update with those new changes. Then we can merge. Let us know if we can help out with that. Good Luck!